api/notd/block_processor.py

```diff
@@ -177,10 +164,68 @@ async def _process_erc721_single_event(self, event: LogReceipt, blockData: Block
         fromAddress = chain_util.normalize_address(event['topics'][1].hex())
         toAddress = chain_util.normalize_address(event['topics'][2].hex())
         tokenId = int.from_bytes(bytes(event['topics'][3]), 'big')
-        ethTransaction = await self._get_transaction(transactionHash=transactionHash, blockData=blockData)
-        operatorAddress = ethTransaction['from']
-        gasLimit = ethTransaction['gas']
-        gasPrice = ethTransaction['gasPrice']
-        value = ethTransaction['value']
-        transactions = [RetrievedTokenTransfer(transactionHash=transactionHash, registryAddress=registryAddress, fromAddress=fromAddress, toAddress=toAddress, operatorAddress=operatorAddress, tokenId=tokenId, value=value, amount=1, gasLimit=gasLimit, gasPrice=gasPrice, blockNumber=blockNumber, tokenType='erc721')]
+        operatorAddress = transaction['from']
+        transactions = [RetrievedEvent(transactionHash=transactionHash, registryAddress=registryAddress, fromAddress=fromAddress, toAddress=toAddress, operatorAddress=operatorAddress, amount=1, tokenId=tokenId, tokenType = 'erc721')]
         return transactions
+
+    @alru_cache()
```

Comment:
is this needed?

Reply:
Well I thought it made thing look neater, but if you want we could remove it

Reply:
more code is always worse and less neat than less code. does it serve an actual purpose here?